docs/organizations/metadata.mdx

```diff
@@ -1,5 +1,92 @@
 ---
-sanity_slug: /docs/organizations/organization-metadata
+title: Organization Metadata
+description: >-
+  Organization objects hold a set of metadata that can be used internally to store arbitrary information.
 ---
 
-# This is a stub
+import { Tables } from "@/components/Table";
```

Comment:
Imports are unnecessary! The docs get rendered by our marketing repo, which is where the components live - they get rendered by the marketing repo.

```suggestion
import { Tables } from "@/components/Table";
```